cwlcon2024-site/index.html

```diff
@@ -31,7 +31,7 @@ <h1 role="doc-subtitle">
         </div>
         <!-- This banner is duplicated below for larger viewports. Please update it there as well. -->
         <div class="banner-text">
-            <span>Week of May 13th, 2024</span>
+            <span>14th - 17th May 2024</span>
```

Comment:
I think we need more words here


The in-person conference proper is only May 15&16th, the optional novice training session is on the 14th (which will have a separate signup), and optional hackathon day is May 17th. Not to mention the online-only sessions the week before. Hmm..

Reply:
Yes I played around with this and nothing seemed to fit nicely.  We could do just 8th - 17th May 2024 but don't want to confuse people before they look at the program

Reply:
Have moved to 8th to 17th